api/provider/aws/task_read.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/quintilesims/layer0/common/models"
 )
 
+// Read returns a Task based on the provided Task ID. The user's active cluster (Environment) is used as a filter
```

Comment:
Ditto re: clarification on `Task`. Is it ours or ECS' concept?

Reply:
This is our concept, since we are [returning a models.Task](https://github.com/quintilesims/layer0/blob/aws-provider-deploy/api/provider/aws/task_read.go#L35). I'll make this adjustment.